client/src/api/merger/merger.api.ts

```diff
@@ -59,6 +60,11 @@ export const getCategories = () =>
   axios
     .get<CategoriesResponse>(`/categories/`)
     .then((response) => response.data);
+    
+export const removeCategory = (data: RemoveCategoryRequest) =>
+  axios
+    .post<CategoriesResponse>(`/categories/remove/`, data)
```

Comment:
I think it's better to make it a `DELETE` request and pass the id in the URL query params